mobile-app/.env.dev.dist

```diff
@@ -0,0 +1 @@
+HEX_API_URL=http://localhost:3000/api
```

Comment:
I think the .env.dev file need to be injected into the expo service in the docker-compose.dev file !

Reply:
Actually the file needed is called `.env` and not `.env.dev` (RTFM buddy!! 🤣 )

And I can confirm it's embedded correctly both with expo build and docker-compose 🙂